src/main/webapp/data/userMapData.json

```diff
@@ -579,6 +604,7 @@
       "Centro de Estudios Superiores de Diseño",
       "CETYS Universidad",
       "Escuela Bancaria y Comercial",
+      "Instituto Cumbres Bosques",
       "Instituto Tecnologico de Estudios Superiores",
       "ITESM",
```

Comment:
Can incorporate this change too: https://github.com/TEAMMATES/teammates/pull/8204/files#r157783084